c7n/resources/rdscluster.py

```diff
@@ -22,6 +22,19 @@
 
 class DescribeCluster(DescribeSource):
 
+    def get_resources(self, ids):
+        """Support server side filtering on arns
+        """
+        if ids[0].startswith('arn:'):
+            filters = [
+                {
+                    'Name': 'db-cluster-id',
+                    'Values': ids
+                },
+            ]
+            params = {'Filters': filters}
```

Comment:
is there a max values re filtering here?

Reply:
Hi @kapilt ,

related the docs:

MaxRecords (integer) --
The maximum number of records to include in the response. If more records exist than the specified MaxRecords value, a pagination token called a marker is included in the response so you can retrieve the remaining results.

Default: 100
Constraints: Minimum 20, maximum 100.